configure.ac

```diff
@@ -746,6 +746,16 @@ case $host in
          dnl It's safe to add these paths even if the functionality is disabled by
          dnl the user (--without-wallet or --without-gui for example).
 
+         dnl Homebrew may create symlinks in /usr/local/include for some packages.
```

Comment:
I believe this would almost be safe enough to do everywhere (not just mac/homebrew), and possibly even for `/usr/include` because it should have no unintended side-effects, and for gcc it will just fail the test.

I'm not suggesting that we do that, just pointing out that it should be pretty much harmless even if it's not needed.

Reply:
I think the wording here is ok; we don't need to call out differences in defaults between x86_64 and arm64 installations of homebrew. Also, just because `/opt/homebrew` is the preferred prefix for arm64, it's not clear to me that a brew installation *couldn't* be in `/usr/local` on arm64, which this would also accomodate.